app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -254,6 +259,11 @@ open class DuckDuckGoApplication : HasAndroidInjector, Application(), LifecycleO
             }
             unsentForgetAllPixelStore.resetCount()
         }
+        submitUnsentFireAppRestartedWithIntentPixels()
+    }
+
+    private fun submitUnsentFireAppRestartedWithIntentPixels() {
+        dataClearerForegroundAppRestartPixel.firePendingPixels()
     }
```

Comment:
Why you create a function just for this?

Reply:
Yeah, why not 🤷‍♀️